monitoring/onpremise/keda/variables.tf

```
@@ -9,12 +9,14 @@ variable "docker_image" {
   description = "Docker image for Keda"
   type = object({
     keda = object({
-      image = string
-      tag   = string
+      registry = string
```

I would mark it optional
```suggestion
      registry = optional(string, "")
```